sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -647,17 +648,17 @@
             "location": "[parameters('location')]"
         },
         {
-            "condition": "[and(variables('enablePE'), equals(parameters('storageAccountOption'), 'new'))]",
+            "condition": "[and(variables('enablePE'), or(equals(parameters('storageAccountOption'), 'new'), and(equals(parameters('set_up_feature_store'), 'true'), empty(parameters('offline_store_target')))))]",
             "type": "Microsoft.Storage/storageAccounts",
             "apiVersion": "2019-04-01",
-            "name": "[parameters('storageAccountName')]",
+            "name": "[variables('provisionedStorageAccountName')]",
             "tags": "[parameters('tagValues')]",
             "dependsOn": [
                 "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('vnetName'), parameters('subnetName'))]"
             ],
-            "location": "[parameters('storageAccountLocation')]",
+            "location": "[if(equals(parameters('set_up_feature_store'), 'false'), parameters('storageAccountLocation'), parameters('location'))]",
             "sku": {
-                "name": "[parameters('storageAccountType')]"
+                "name": "[if(equals(parameters('set_up_feature_store'), 'false'), parameters('storageAccountType'), 'Standard_LRS')]"
```

Comment:
same here. Change the condition to equals(xxx, 'true')

Reply:
done